Code/GraphMol/RGroupDecomposition/RGroupMatch.h

```diff
@@ -17,6 +17,7 @@ typedef std::map<int, RData> R_DECOMP;
 
 //! RGroupMatch is the decomposition for a single molecule
 struct RGroupMatch {
+ public:
```

Comment:
this is redundant: members of a struct are by default public

Reply:
I know, it was just for better clarity, but I am happy to remove it if you prefer so.

Reply:
I have removed it - I actually think it was a left-over from a previous incarnation where I had added a `private` method that I have subsequently removed.

Reply:
> I know, it was just for better clarity, but I am happy to remove it if you prefer so.

I don't think we need to add clarifications for basic language features like "members of structs are public by default"